src/helpers/validators.ts

```diff
@@ -37,13 +37,12 @@ export const isValidEmail = (email: any) =>
 export const isENSAddressFormat = memoFn(address => {
   // @ts-expect-error ts-migrate(2571) FIXME: Object is of type 'unknown'.
   const parts = !!address && address.split('.');
+
   if (
     !parts ||
     parts.length === 1 ||
     !parts[parts.length - 1] ||
-    (parseDomain(parts[parts.length - 1].toLowerCase()).type ===
-      ParseResultType.NotListed &&
-      parts[parts.length - 1].toLowerCase() !== 'eth') ||
+    parts[parts.length - 1].toLowerCase() !== 'eth' ||
```

Comment:
Here I simplified the logic because in the previous form it was allowing addresses ending with `.et` because it is a valid country domain and it was breaking e2e and generally causes a flash in the UI.

Reply:
We don't need to add it to this PR, but this is a good example of something that should have a unit test